src/main/java/com/alyrow/gdx/particle/utils/T.java

```diff
@@ -0,0 +1,322 @@
+package com.alyrow.gdx.particle.utils;
+
+import java.awt.*;
+import java.awt.geom.AffineTransform;
+import java.awt.geom.NoninvertibleTransformException;
+import java.awt.geom.Point2D;
+
+public class T<Holder extends T.Transformer> {
+
+    private AffineTransform t;
```

Comment:
Seems that the equivalent is `Affine2`, but it also seems that method like `getType(), getMatrix(), ...` will be difficult to have.